Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -399,6 +559,171 @@ def get_scan_status_command():
     }
     return get_entry_for_object('Scan status for {}'.format(scan_id), 'TenableIO.Scan(val.Id === obj.Id)', scan_status)
 
+# TEST
+
+
+def pause_scan_command():
+    """Function for integration command pause_scan_command."""
+    scan_id = demisto.getArg('scanId')
+    scan_details = send_scan_request(scan_id)
+    scan_status = {
+        'Id': scan_id,
+        'Status': scan_details['info']['status']
+    }
+
+    if scan_status["Status"].lower() == "running":
+        send_scan_request(scan_id, "pause", "POST")
+        paused_scan = {
+            "Id": scan_id,
+            "Status": "Pausing"
+        }
+
+        return get_entry_for_object(
+            "The requested scan was paused successfully", "TenableIO.Scan", replace_keys(paused_scan), ["Id", "Status"])
+
+    else:
+        return "Command 'tenable-io-pause-scan' cannot be called while scan status is {}".format(scan_status["Status"])
+
+
+def pause_scans_command():
```

Comment:
Please add the logic you have in `pause_scans_command` to `pause_scan_command` and delete  `pause_scans_command`

Reply:
Done
Update 02.23.2021 1034 HRS EST:
Removed command from code, and added the functionality to Pause_scan_comman